asset/scss-about/_queries.scss

```diff
@@ -0,0 +1,240 @@
+@media screen and (min-width: 769px) {
+  .mobile_version {
+    display: none;
+  }
+
+  .header_text {
+    width: 100vw;
+  }
+}
+
+@media screen and (min-width: 769px) {
```

Comment:
- [ ] Kindly notice that lines 1 and 11 are exactly the same. Please apply the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) (Don't Repeat Yourself) rule and remove/refactor any duplicated code.